src/Bids/StandardModernPrecision/Lampe.hs

```diff
@@ -31,6 +155,22 @@ shapeShower2S = do
     makeAlertableCall (T.Bid 2 T.Spades) "5-3 in the minors, either way"
 
 
+_shapeShower2S2N :: Action
```

Comment:
Starting a value with an underscore turns off the linter warnings about unused values. Rename so it _ends_ with an underscore instead.